addons/packages/ako-operator/1.5.0/README.md

```diff
@@ -29,15 +29,20 @@ The following configuration values can be set to customize the ako-operator inst
 | `avi_password` | Required | Avi controller password. |
 | `avi_cloud_name` | Required | the configured cloud name on the Avi controller. |
 | `avi_service_engine_group` | Required | the group name of Service Engine that's to be used by the set of AKO Deployments. |
+| `avi_management_cluster_service_engine_group` | Required | the group name of Service Engine that's to be used by the AKO in management cluster. |
 | `avi_data_network` | Required | describes the Data Networks the AKO will be deployed with. |
 | `avi_data_network_cidr` | Required | describes the Data Networks the AKO will be deployed with. |
+| `avi_cp_network` | Required | describes the Control Plane Networks of the workload cluster. |
+| `avi_cp_network_cidr` | Required | describes the Control Plane Networks of the workload cluster. |
 | `avi_ca_data_b64` | Required | Avi controller credential. |
 | `avi_labels` | Optional | Label used to select Clusters. The Clusters that are selected by this will be the ones affected by this AKODeploymentConfig. |
 | `avi_cni_plugin` | Optional | describes which cni plugin cluster is using. AKO supported CNI: antrea,calico,canal,flannel,openshift and ncp |
 | `avi_disable_static_route_sync` | Optional | describes ako should sync static routing or not. |
 | `avi_control_plane_ha_provider` | Required | describes whether Avi provides control plane HA service or not. |
 | `avi_management_cluster_vip_network_name` | Required | describes the data network name of the management cluster AKO will be deployed with. |
 | `avi_management_cluster_vip_network_cidr` | Required | describes the data network cidr of the management cluster AKO will be deployed with. |
+| `avi_management_cluster_cp_vip_network_name` | Required | describes the name of the management cluster control plane network. |
```

Comment:
Can we avoid use `cp`? I think we can use `control_plane`, don't be afraid the name is long, we need it to be clear.